CHANGELOG.md

```diff
@@ -1,5 +1,98 @@
+  make the conversation update code paths more uniform, and also reduce special
+  cases for local and remote objects. (#1801)
+* Merged http2-client fixes as mentioned in the comments of #1703 (#1809)
+* Some executables now have a runtime dependency on ncurses (#1791)
+* Minor changes around SAML and multi-team Issuers.
+  - Change query param to not contain `-`, but `_`.  (This is considered an internal change because the feature has been release in the last release, but only been documented in this one.)
+  - Haddocks.
+  - Simplify code.
+  - Remove unnecessary calls to cassandra. (#1763)
+* Clean up JSON Golden Tests (Part 6) (#1769)
+* Remove explicit instantiations of ErrorDescription (#1794)
+* Remove one flaky integration test about ordering of search results (#1798)
+* Report all failures in JSON golden tests in a group at once (#1746)
+* Convert the `PUT /conversations/:cnv/access` endpoint to Servant (#1807)
+* Move /connections/* endpoints to Servant (#1770)
+* Servantify Galley's DELETE /i/user endpoint (#1772)
+* Convert the `PUT /conversations/:cnv/message-timer` endpoint to Servant (#1780)
+* Convert the `PUT /conversations/:cnv/members/:usr` endpoint to Servant (#1796)
+* Convert the `PUT /conversations/:cnv/receipt-mode` endpoint to Servant (#1797)
+* Expose wire.com internal EJDP process to backoffice/stern. (#1831)
+* Update configurable boolean team feature list in backoffice/stern. (#1829)
+* Handle upper/lower case more consistently in scim and rich-info data. (#1754)
+
+## Federation changes
+
+* Add value for verification depth of client certificates in federator ingress (#1812)
+* Document federation API conventions and align already existing APIs (#1765)
+* Notify remote users when a conversation access settings are updated (#1808)
+* Notify remote users when a conversation member role is updated (#1785)
+* Notify remote users when a conversation message timer is updated (#1783)
+* Notify remote users when a conversation is renamed (#1767)
+* Make sure that only users that are actually part of a conversation get notified about updates in the conversation metadata (#1767)
+* Notify remote users when a conversation receipt mode is updated (#1801)
+* Implement updates to remote members (#1785)
+* Make conversation ID of the on-conversation-created RPC unqualified (#1766)
+* 4 endpoints for create/update/get/list connections designed for remote users in mind. So far, the implementation only works for local users (actual implementation will come as a follow-up) (#1773)
+* The returned `connection` object now has a `qualified_to` field with the domain of the (potentially remote) user. (#1773)
+* Add migration for remote connection table (#1789)
+* Remove a user from remote conversations upon deleting their account (#1790)
+* Remove elasticsearch specific details from the search endpoint (#1768)
+* Added support for updating self member status of remote conversations (#1753)
+
```

Comment:
```suggestion
```

Reply:
I disagree!  :)  Also, there are two empty lines between earlier releases.  Having that said, I would have taken the change, but didn't want to wait for CI again.